### ISSUE TITLE
Pagination on "Sync ratings" endpoints is not possible

### ISSUE THREAD

**Issue by defhead** [UweTrottmann/trakt-java#123]
Hello,

Trakt API allows to paginate through "[GET] /sync/ratings/<type>" endpoint but it's not possible to accomplish that using trakt-java API. Theses methods take only "rating" and "extended" parameters:

```
    @GET("sync/ratings/movies{rating}")
    Call<List<RatedMovie>> ratingsMovies(
            @Path(value = "rating", encoded = true) RatingsFilter filter,
            @Query(value = "extended", encoded = true) Extended extended
    );

    @GET("sync/ratings/shows{rating}")
    Call<List<RatedShow>> ratingsShows(
            @Path(value = "rating", encoded = true) RatingsFilter filter,
            @Query(value = "extended", encoded = true) Extended extended
    );

    @GET("sync/ratings/seasons{rating}")
    Call<List<RatedSeason>> ratingsSeasons(
            @Path(value = "rating", encoded = true) RatingsFilter filter,
            @Query(value = "extended", encoded = true) Extended extended
    );

    @GET("sync/ratings/episodes{rating}")
    Call<List<RatedEpisode>> ratingsEpisodes(
            @Path(value = "rating", encoded = true) RatingsFilter filter,
            @Query(value = "extended", encoded = true) Extended extended
    );
```

Is there an explanation for that?

Thank you.

**Comment by UweTrottmann** [comment]
IIRC pagination is optional and has been added to existing methods? Feel free to submit a PR that adds pagination support (look at other methods that already support it).

**Comment by defhead** [comment]
You can close this issue, it was fixed by #124 .
